tools/add_column.rb

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
+
+require 'csv'
+
+# Parse options
+if ARGV.length == 0 || (ARGV.length == 1 && ARGV[0] == '-h')
+  puts "Usage: ./tools/add_column.rb info:en-us --after=info:tr --sameas=info:en < stations.csv > stations.csv"
+  puts "       ./tools/add_column.rb ntv_is_enabled --content=f < stations.csv > stations.csv"
+  puts "       ./tools/add_column.rb carrier_id '--contentprog=row[3] if [\"8775561\"].include?(row[3])' < stations.csv > stations.csv"
+  exit
+end
+
+csv = CSV.parse(STDIN, headers: true, col_sep: ";", encoding: "UTF-8")
```

Comment:
maybe put the parameters in a variable, to be sure to use the same in the output